ui/addon/src/main/java/org/jboss/windup/ui/RulesetUpdateChecker.java

```diff
@@ -54,8 +57,7 @@ public void perform(@Observes PostStartup event)
         }
     }
 
-    public static boolean rulesetNeedUpdate(DependencyResolver dependencyResolver)
-    {
+    public static boolean rulesetNeedUpdate(DependencyResolver dependencyResolver, String rulesPomXml) {
         List<Coordinate> resolveVersions = dependencyResolver.resolveVersions(DependencyQueryBuilder.create(CoordinateBuilder.create()
```

Comment:
This should probably be wrapped in an Environment.isOffline() check from the `org.jboss.forge.addon:environment` addon.